### PR TITLE
[TechDraw] Improve weld task tool tips

### PR DESCRIPTION
**Pull request by benj5378**
Improve weld task tool tips, partially fixes #0004838.

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [x]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [x]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [x]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [x]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [x]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

File: src/Mod/TechDraw/Gui/TaskWeldingSymbol.ui [hunk 1]
```diff
@@ -125,14 +128,16 @@
...
+Angle, surface finish, root</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="leOtherTextR">
            <property name="toolTip">
-            <string>Text after other side symbol</string>
+            <string>Text after other side symbol
+No. of welds × length, (gap)</string>
```
Comment (luzpaz) [inline src/Mod/TechDraw/Gui/TaskWeldingSymbol.ui:140]: `No.` maybe confusing to translators. Better to just write out the word.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5475">pipeline status</a> for feature branch [PR_5475](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5475). Pipeline [460535358](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/460535358) was triggered at [9724a44](https://github.com/FreeCAD/FreeCAD/pull/5475/commits/9724a44c6aa4fbef87e1af6a0c16aa62be19f2d9). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
Please also prepend `[TD}` the commit title before merge, thanks!

**Comment by WandererFan** [comment]
Can I merge this?

**Comment by luzpaz** [comment]
@WandererFan if OP doesn't respond, then I'd suggest manually changing the 2 places where `No.` is mentioned in the commit to `Number` and also amending the commit summary title by prepending `[TD]` to it

**Comment by WandererFan** [comment]
> @WandererFan if OP doesn't respond, then I'd suggest manually changing the 2 places where `No.` is mentioned in the commit to `Number` and also amending the commit summary title by prepending `[TD]` to it

Is there a trick to do this within the PR/Github world?  Or do I have to do the traditional pull/push?

**Comment by chennes** [comment]
I have made my own branch, cherry picked onto it, and made the changes to the cherry-picked commits. There's probably a better way, but that gets the job done anyway.

**Comment by luzpaz** [comment]
Yes, you need to: 
1. Make the appropriate tooltip changes  
2. Commit them  
3. Squash everything down in to one commit
4. Amend said commit summary to have [TD] prepended
5. git push (force) to overwrite the previous changes in the `benj5378:tiptools2` branch

File: src/Mod/TechDraw/Gui/TaskWeldingSymbol.ui (hunk 1 above)
Comment (luzpaz) [reply src/Mod/TechDraw/Gui/TaskWeldingSymbol.ui]: Addressed in 86aed65

**Comment by luzpaz** [comment]
```
         <string>Directory to welding symbols.
This directory will be used for the symbol selection.</string>
```
I don't understand this tooltip. Shouldn't it be: 
```
         <string>Welding symbols directory path. This directory will be 
used for the symbol selection.</string>
```
ref: https://github.com/FreeCAD/FreeCAD/blob/86aed6581e98a3e04948446e243d2bddc996265a/src/Mod/TechDraw/Gui/TaskWeldingSymbol.ui#L274

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5475">pipeline status</a> for feature branch [PR_5475](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5475). Pipeline [478298448](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/478298448) was triggered at [86aed65](https://github.com/FreeCAD/FreeCAD/pull/5475/commits/86aed6581e98a3e04948446e243d2bddc996265a). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
Any feedback?

**Comment by WandererFan** [comment]
Sorry, I'm travelling.   Your version of the tip is better than the
original.

**Comment by luzpaz** [comment]
@WandererFan I'll push another commit. I'll skip the CI, no need to run a test on a trivial fix. Then you can merge when you have a moment?

**Comment by luzpaz** [comment]
@wandererfan I just went ahead and pushed another commit in [1cbea2a](https://github.com/FreeCAD/FreeCAD/pull/5475/commits/1cbea2a7eb51bb03444bc5a5642a2f0d980c6d13). I refined the tooltip further + skipped the ci

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5475">pipeline status</a> for feature branch [PR_5475](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5475). Pipeline [478983767](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/478983767) was triggered at [1cbea2a](https://github.com/FreeCAD/FreeCAD/pull/5475/commits/1cbea2a7eb51bb03444bc5a5642a2f0d980c6d13). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
Thanks for merging. FYI, I was hoping you'd squash them after you reviewed it. Next time I'll squash them JIC.

**Comment by WandererFan** [comment]
Can this be done within the Github/PR process or only in a local repo?

**Comment by luzpaz** [comment]
Yes, within the GH UI you can choose 'Squash and Merge' and then modify the commit message (to include the squashed commit summaries or not). Again, through the GH UI if necessary.

**Pull request merged**